modules/peddy/main.nf

```diff
@@ -0,0 +1,49 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process PEDDY {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::peddy=0.4.8" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/peddy:0.4.8--pyh5e36f6f_0"
+    } else {
+        container "quay.io/biocontainers/peddy:0.4.8--pyh5e36f6f_0"
+    }
+
+    input:
+    tuple val(meta), path(vcf)
+    tuple val(meta), path(vcf_tbi)
```

Comment:
```suggestion
    tuple val(meta), path(vcf), path(vcf_tbi)
```

Reply:
These should be in the same channel so the index and the main file are together.